lib/src/replicache.dart

```diff
@@ -539,7 +539,7 @@ class Replicache implements ReadTransaction {
     R rv;
     try {
       final tx = newWriteTransaction(this._invoke, txId);
-      rv = await Function.apply(mutatorImpl, [tx, args]);
+      rv = await mutatorImpl(tx, args);
```

Comment:
`Function.apply` was a remnant of the past when I though we could provide a richer interface to the mutator functions.